hub/executors/encoders/image/torchvision-densenet161/torchvision_densenet.yml

```diff
@@ -0,0 +1,8 @@
+!TorchVisionDenseNet161
```

Comment:
```suggestion
!ImageTorchEncoder
```

Reply:
Understood and making necessary changes.
Shall I also port remaining executors to hub?
Post this, I'll start adding GAN models.